access/grpc/grpc.go

```diff
@@ -165,13 +165,19 @@ func (c *BaseClient) GetNodeVersionInfo(ctx context.Context, opts ...grpc.CallOp
 	}
 
 	info := res.GetInfo()
+	compRange := flow.CompatibleRange{
```

Comment:
it's possible for `CompatibleRange` to be nil if the version control isn't configured. let's make it a pointer here as well.